parsers/BR.py

```diff
@@ -1,191 +1,184 @@
+    for k in entries_to_remove:
         totals.pop(k, None)
 
-    mapped_totals = {generation_mapping.get(name, 'unknown'): val for name, val
-                     in totals.items()}
+    mapped_totals = {
+        GENERATION_MAPPING.get(name, "unknown"): val for name, val in totals.items()
+    }
 
     return dt, mapped_totals
 
 
 def fetch_production(zone_key, session=None, target_datetime=None, logger=None) -> dict:
     """Requests the last known production mix (in MW) of a given country."""
     if target_datetime:
-        raise NotImplementedError('This parser is not yet able to parse past dates')
+        raise NotImplementedError("This parser is not yet able to parse past dates")
 
     gd = get_data(session, logger)
     generation = production_processor(gd, zone_key)
 
     datapoint = {
-      'zoneKey': zone_key,
-      'datetime': generation[0].datetime,
-      'production': generation[1],
-      'storage': {
-          'hydro': None,
-      },
-      'source': 'ons.org.br'
+        "zoneKey": zone_key,
+        "datetime": generation[0].datetime,
+        "production": generation[1],
+        "storage": {
+            "hydro": None,
+        },
+        "source": SOURCE,
     }
 
-    datapoint = validate(datapoint, logger,
-                         remove_negative=True, required=['hydro'], floor=1000)
+    datapoint = validate(
+        datapoint, logger, remove_negative=True, required=["hydro"], floor=1000
+    )
 
     return datapoint
 
 
-def fetch_exchange(zone_key1, zone_key2, session=None, target_datetime=None, logger=None) -> dict:
+def fetch_exchange(
+    zone_key1, zone_key2, session=None, target_datetime=None, logger=None
+) -> dict:
     """Requests the last known power exchange (in MW) between two regions."""
     if target_datetime:
-        raise NotImplementedError('This parser is not yet able to parse past dates')
+        raise NotImplementedError("This parser is not yet able to parse past dates")
 
     gd = get_data(session, logger)
 
-    if zone_key1 in countries_exchange.keys():
-        country_exchange = countries_exchange[zone_key1]
+    if zone_key1 in COUNTRIES_EXCHANGE.keys():
+        country_exchange = COUNTRIES_EXCHANGE[zone_key1]
 
-    if zone_key2 in countries_exchange.keys():
-        country_exchange = countries_exchange[zone_key2]
+    if zone_key2 in COUNTRIES_EXCHANGE.keys():
+        country_exchange = COUNTRIES_EXCHANGE[zone_key2]
 
-    data = {
-        'datetime': arrow.get(gd['Data']).datetime,
-        'sortedZoneKeys': '->'.join(sorted([zone_key1, zone_key2])),
-        'netFlow': gd['internacional'][country_exchange['name']] * country_exchange['flow'],
-        'source': 'ons.org.br'
+    return {
+        "datetime": arrow.get(gd["Data"]).datetime,
+        "sortedZoneKeys": "->".join(sorted([zone_key1, zone_key2])),
+        "netFlow": gd["internacional"][country_exchange["name"]]
+        * country_exchange["flow"],
```

Comment:
This formatting looks weird. Can we pull it out into a variable, like in `fetch_region_exchange`?